www/src/templates/tags.js

```diff
@@ -16,52 +17,26 @@ const Tags = ({ pageContext, data, location }) => {
     <Layout location={location}>
       <Container>
         <h1>{tagHeader}</h1>
-        <ul>
-          {edges.map(({ node }) => {
-            const {
-              frontmatter: { title },
-              fields: { slug },
-            } = node
-            return (
-              <li key={slug}>
-                <Link to={slug}>{title}</Link>
-              </li>
-            )
-          })}
-        </ul>
+        {edges.map(({ node }) => (
+          <BlogPostPreviewItem
+            post={node}
+            key={node.fields.slug}
+            css={{ marginBottom: rhythm(2) }}
+          />
+        ))}
         <Link to="/blog/tags">All tags</Link>
       </Container>
     </Layout>
   )
 }
 
-Tags.propTypes = {
-  pageContext: PropTypes.shape({
-    tag: PropTypes.string.isRequired,
-  }),
-  data: PropTypes.shape({
-    allMarkdownRemark: PropTypes.shape({
-      totalCount: PropTypes.number.isRequired,
-      edges: PropTypes.arrayOf(
-        PropTypes.shape({
-          node: PropTypes.shape({
-            frontmatter: PropTypes.shape({
-              title: PropTypes.string.isRequired,
-            }),
-          }),
-        }).isRequired
-      ),
-    }),
-  }),
-}
-
 export default Tags
 
 export const pageQuery = graphql`
   query($tag: String) {
     allMarkdownRemark(
       limit: 2000
-      sort: { fields: [frontmatter___date, fields___slug], order: DESC }
+      sort: { fields: [frontmatter___date], order: DESC }
```

Comment:
Why remove this sort field? This makes sure we don't have potentially inconsistencies between builds when there are multiple blog posts from same day/date (sometimes one post can be before the other and in next build it might change - depending on order of node creation).

Reply:
(I know it's pretty fringe issue, but please, let's not remove it)